bin/check-workflow-run.js

```diff
@@ -8,8 +8,11 @@
 const Github = require('./github')
 
 const SUCCESS_MSG = '*** [OK] ***'
-const WORKFLOWS = ['ci-workflow.yml']
-const AGENT_REPO = 'node-newrelic'
+
+async function filterAsync(array, cb) {
+  const filterMap = await Promise.all(array.map(cb))
```

Comment:
With the name `cb` I had to walk the code to understand what was happening here. Wonder if a more specific variable name would help. `checkWorkflowSuccess` maybe?

Not critical but food for thought...

Reply:
sure I can do that. You caught me, SO rip off and didn't think much of it